docs/docsource/operation/Introduction/Conceptual_Framework.rst

```diff
@@ -1,30 +1,45 @@
-services) store data as files in buckets. Administrators are free to
-configure and name buckets as they please—to the admin, buckets are
-essentially tenants aggregated in a cloud storage deployment. From an
-end user’s perspective, a bucket simply appears as a networked storage
-location, for example, “accounting,” “bioinformatics-lab,”
-“daily-video-backup,” or any other organizational unit that makes
-sense.
+Fundamental building blocks of Zenko’s replication framework are the bucket, the
+location, and the cloud service. End users (people or services) store data as
+files in buckets. Administrators are free to configure and name buckets as they
+please—to the admin, buckets are essentially tenants aggregated in a cloud
+storage deployment. From an end user’s perspective, a bucket simply appears as a
+networked storage location, for example, “accounting,” “bioinformatics-lab,”
+“daily-video-backup,” or any other organizational unit that makes sense.
 
-Zenko fetches buckets from one location—a Scality RING private cloud,
-for example—and replicates them to designated cloud storage locations on
-other clouds. So, a company might store its data by departmental buckets
-to local cloud, then replicate that cloud storage location to several
-other public or private clouds for storage redundancy, rate-shopping, or
-other purposes. Zenko manages the complexities and discontinuities
-between the native Amazon S3 cloud storage protocol, and other popular
-cloud protocols, such as Google Cloud and Microsoft Azure storage, and
-manages a unified namespace to retrieve data seamlessly from all managed
-cloud locations.
+Zenko can fetch buckets from one location—a Scality RING private cloud, for
+example—and replicate them to designated cloud storage locations on other
+clouds. A company may, for example, store its data by departmental buckets to
+local cloud, then replicate that cloud storage location to several other public
+or private clouds for storage redundancy, rate-shopping, or other
+purposes. Zenko can also provide useful management features for large quantities
+of unstructured data without replicating data at all: a company with very large
+in-house data stores may not replicate using Zenko, but may still want a robust
+metadata search capability to locate files or file versions rapidly.
 
-In addition to replication, Zenko provides bucket lifecycle management
-(expiration and transition) and other tools to manage data easily in a
-multi-cloud environment.
\ No newline at end of file
+Zenko manages the complexities and discontinuities between the native Amazon S3
+cloud storage protocol and other popular cloud protocols, such as Google Cloud
+and Microsoft Azure storage, and manages a unified namespace to retrieve data
+seamlessly from all managed cloud locations.
+
+To do this, Zenko maintains its own namespace, which it uses to manage and track
+objects in other namespaces. For example, a cloud might hold a bucket in a dozen
```

Comment:
I'm not sure I understand `a cloud might hold a bucket in a dozen places`

Reply:
if I change "hold a bucket in a dozen places" to "distribute a bucket's contents over several disks." does that work?

Reply:
Ah, I see your intent now. Yeah either that or something like "a cloud might spread a bucket's contents across several servers".  Either way gets the point across.